CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Changelog
 
+## [v2.4.2](https://github.com/bandprotocol/chain/releases/tag/v2.4.2)
```

Comment:
Please change to unreleased (We will update this changelog before cut release later)

Reply:
but this changelog for this version will contain only this change